File: 백준/Silver/11652. 카드/카드.java
import java.util.*;
import java.lang.*;
import java.io.*;

class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        
        int N = Integer.parseInt(br.readLine());
        
        List<Long> list = new ArrayList<>();
        for(int i=0; i<N; i++) list.add(Long.parseLong(br.readLine()));
        
        Collections.sort(list);
        
        Long answer = list.get(0);
        int max_cnt = 0;
        int cnt = 1;
        Long cur = list.get(0);
        
        for(int i=1; i<list.size(); i++) {
            if(cur.equals(list.get(i))) cnt++;
            else {
                if(max_cnt < cnt) {
                    max_cnt = cnt;
                    answer = cur;
                }
                cur = list.get(i);
                cnt = 1;
            }
        }
        if(max_cnt < cnt) answer = cur;
        
        System.out.println(answer);
    }
}

File: 백준/Silver/11652. 카드/카드.swift
let N = Int(readLine()!)!
var arr:[Int] = []
for _ in 0..<N { arr.append(Int(readLine()!)!) }
arr.sort()

var answer = 0
var max_cnt = 0
var cnt = 1
var cur = arr[0]

for i in 1..<N {
    if(cur == arr[i]) { cnt += 1 }
    else {
        if(max_cnt < cnt) {
            answer = cur
            max_cnt = cnt
        }
        cnt = 1
        cur = arr[i]
    }
}
if(max_cnt < cnt) { answer = cur }

print(answer)